core-services/src/test/java/org/zalando/nakadi/service/publishing/check/EnrichmentCheckTest.java

```diff
@@ -44,7 +44,7 @@ public void testExecuteSuccess() {
         final NakadiRecord nakadiRecord = Mockito.mock(NakadiRecord.class);
         final List<NakadiRecord> nakadiRecords = List.of(nakadiRecord);
 
-        assertNull(enrichmentCheck.execute(eventType, nakadiRecords));
+        assertNotNull(enrichmentCheck.execute(eventType, nakadiRecords));
```

Comment:
Should we have a more elaborate assertion here?  Not `null` can still communicate both success (empty list) and failure (non-empty list).